src/wr_logic_teleop_arm/src/SolenoidController.cpp

```diff
@@ -0,0 +1,64 @@
+#include "SolenoidController.hpp"
+#include "ros/node_handle.h"
+#include "ros/subscriber.h"
+#include "std_msgs/Bool.h"
+
+#include <fcntl.h>
+#include <fstream>
+
+
+
+constexpr uint32_t MESSAGE_QUEUE_LENGTH = 1000;
```

Comment:
Should not probably be 1000:
```suggestion
constexpr uint32_t MESSAGE_QUEUE_LENGTH{1};
```

Reply:
Fixed in 39a81c1f.